Code/GraphMol/FileParsers/MolFileWriter.cpp

```diff
@@ -1155,7 +1156,74 @@ void appendEnhancedStereoGroups(std::string &res, const RWMol &tmol) {
     res += "M  V30 END COLLECTION\n";
   }
 }
+namespace FileParserUtils {
+std::string getV3000CTAB(const ROMol &tmol, int confId) {
+  auto nAtoms = tmol.getNumAtoms();
+  auto nBonds = tmol.getNumBonds();
+  const auto &sgroups = getSubstanceGroups(tmol);
+  auto nSGroups = sgroups.size();
+
+  unsigned chiralFlag = 0;
+  tmol.getPropIfPresent(common_properties::_MolFileChiralFlag, chiralFlag);
+
+  const Conformer *conf = nullptr;
+  if (confId >= 0 || tmol.getNumConformers()) {
+    conf = &(tmol.getConformer(confId));
+  }
+
+  std::string res = "M  V30 BEGIN CTAB\n";
+  std::stringstream ss;
+  ss << "M  V30 COUNTS " << nAtoms << " " << nBonds << " " << nSGroups << " 0 "
+     << chiralFlag << "\n";
+  //      |
+  //      num3DConstraints (not implemented)
```

Comment:
Github's formatting might be fooling me, but this doesn't look like it's pointing at the " 0 " field above anymore. We should probably update this comment to not rely on the formatting.